internal/converter/profiles/converter_profiles.go

```diff
@@ -49,6 +49,7 @@ func New() Profiles {
 		{"KoF", "Kobo Forma", 1440, 1920},
 		{"KoS", "Kobo Sage", 1440, 1920},
 		{"KoE", "Kobo Elipsa", 1404, 1872},
+		{"RM2", "reMarkable 2", 1404, 1872},
```

Comment:
reMarkable is a Kobo device ?
if not, can you add a comment like
```
// Other
```
or 
```
// reMarkable
```

Reply:
Added `// reMarkable`

There's also a reMarkable 1, but i think they are quite rare and a version 3 is supposed to come out next year. So we could still add those later.

Reply:
ok